src/golog_adapter/include/golog_adapter/app.h

```diff
@@ -0,0 +1,56 @@
+/***************************************************************************
+ *  app.h - The tool's main app library of the GOLOG variety
```

Comment:
This seems to be a duplicate of `golog_app/include/golog_app/app.h`

Reply:
Oh this happened probably during my re-naming. Good catch!